packages/lib/src/interactions/SelectionTool.tsx

```diff
@@ -125,6 +125,17 @@ function SelectionTool(props: Props) {
     [],
     { event: 'keydown' }
   );
+  useKeyboardEvent(
+    'Escape',
+    () => {
+      if (!modifierKey) {
```

Comment:
Does this condition matter? Just wondering.

Reply:
Not really. We could have the disable by `Escape` even if `modifierKey` is defined.